ui/admin/app/templates/scopes/scope/sessions/index.hbs

```diff
@@ -61,9 +61,9 @@
               >{{t 'actions.clear-all-filters'}}
               </Rose::Button>
             {{/if}}
-            <LoadingButton @onClick={{route-action 'refreshSessions'}}>
-              {{t 'actions.refresh'}}
-            </LoadingButton>
+
```

Comment:
Should there be whitespace here? Does `prettier` check this?

Reply:
Prettier did not seem to mind for me. I was following suit with what was already in place when it was added to Sessions in desktop. As well I think it helps bring attention to a single line of code that is easily missed without the whitespace. However, I am open to removing it.